cmd/quickStart/quickStart.go

```diff
@@ -0,0 +1,306 @@
+package quickStart
+
+import (
+	"archive/zip"
+	"errors"
+	"fmt"
+	"github.com/ahmetb/go-linq/v3"
+	"github.com/armory/armory-cli/pkg/cmdUtils"
+	"github.com/armory/armory-cli/pkg/config"
+	"github.com/armory/armory-cli/pkg/org"
+	"github.com/armory/armory-cli/pkg/util"
+	"github.com/manifoldco/promptui"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+type GithubQuickStartProject struct {
+	ProjectName   string
+	BranchName    string
+	IsZipFile     bool
+	DirName       string
+	DeployYmlName string
+}
+
+func (p GithubQuickStartProject) GetSuffix() string {
+	if p.IsZipFile {
+		return githubZipSuffix
+	} else {
+		return ""
+	}
+}
+
+func (p GithubQuickStartProject) GetUrl() string {
+	return fmt.Sprintf("%s%s%s", githubBaseUrl, p.ProjectName, p.GetSuffix())
+}
+
+func (p GithubQuickStartProject) GetProjectFolder() string {
+	return fmt.Sprintf("%s%s-%s", string(os.PathSeparator), p.ProjectName, p.BranchName)
+}
+
+func (p GithubQuickStartProject) GetFileDownloadPath() string {
+	suffix := ""
+	if p.IsZipFile {
+		suffix = ".zip"
+	}
+	return fmt.Sprintf("%s%s", p.DirName, suffix)
+}
+
+func (p GithubQuickStartProject) GetDeployCommand() string {
+	return fmt.Sprintf("armory deploy start -f %s", p.DeployYmlName)
+}
+
+func (p GithubQuickStartProject) Unzip() error {
+
+	if !p.IsZipFile {
+		return nil
+	}
+	log.Info("Unzipping project...")
+	archive, err := zip.OpenReader(p.GetFileDownloadPath())
+
+	if err != nil {
+		return err
+	}
+	defer archive.Close()
+	for _, f := range archive.File {
+		if strings.Contains(f.Name, string(os.PathSeparator)+".") {
+			log.Debugln(fmt.Sprintf("skipping hidden file %s", f.Name))
+			continue
+		}
+		path := filepath.Join(p.DirName, f.Name)
+		dstPath := strings.Replace(path, p.GetProjectFolder(), "", 1)
+
+		if !strings.HasPrefix(path, filepath.Clean(p.DirName)+string(os.PathSeparator)) {
+			return errors.New("found an invalid file path")
+		}
+		if f.FileInfo().IsDir() {
+			log.Debugln(dstPath)
+			os.MkdirAll(dstPath, os.ModePerm)
+			continue
+		}
+
+		log.Debugln(dstPath)
+
+		if err := os.MkdirAll(filepath.Dir(dstPath), os.ModePerm); err != nil {
+			return err
+		}
+
+		dstFile, err := os.OpenFile(dstPath, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, f.Mode())
+		if err != nil {
+			return err
+		}
+
+		fileInArchive, err := f.Open()
+		if err != nil {
+			return err
+		}
+
+		if _, err := io.Copy(dstFile, fileInArchive); err != nil {
+			return err
+		}
+
+		dstFile.Close()
+		fileInArchive.Close()
+	}
+
+	return nil
+}
+
+/**
+Returns isCancelled, error
+*/
+func (p GithubQuickStartProject) Download() (bool, error) {
+	log.Info(fmt.Sprintf("Downloading demo project from `%s`...", p.GetUrl()))
```

Comment:
```suggestion
	log.Info(fmt.Sprintf("Downloading sample application from `%s`...", p.GetUrl()))
```
I checked with Stephen, and he'd like "sample application"